src/main/kotlin/com/avail/descriptor/bundles/MessageBundleTreeDescriptor.kt

```diff
@@ -393,14 +413,14 @@ class MessageBundleTreeDescriptor private constructor(
 		val bundleCount = allPlansInProgress.mapSize()
 		if (bundleCount <= 15) {
 			val strings: MutableMap<String, Int> = HashMap(bundleCount)
-			allPlansInProgress.forEach(BiConsumer { _, value: A_Map ->
-				value.forEach(BiConsumer { _, plansInProgress: A_Set ->
+			allPlansInProgress.forEach { _, value: A_Map ->
+				value.forEach { _, plansInProgress: A_Set ->
 					plansInProgress.forEach { planInProgress ->
 						val string = planInProgress.nameHighlightingPc()
 						strings[string] = strings.getOrDefault(string, 0) + 1
 					}
-				})
-			})
+				}
+			}
```

Comment:
On the next line, please replace with mutableListOf().

Reply:
```kt
			val sorted = mutableListOf<String>()
```